operator/operator_test.go

```diff
@@ -122,18 +122,23 @@ func initEnvironment(t *testing.T) *environment {
 	var cmd *exec.Cmd
 	runGanache := true
 	if runGanache {
```

Comment:
Please boyscout-fix this. Either just remove the bool and execute the indented block directly or pass `runGanache bool` as a boolean flag to `initEnvironment`. Not sure if there's a use case where we don't want to start it though. @matthiasgeihs what was the intention here?

Reply:
That would actually have been much easier than to inject the spawn command via an `env` variable...  
I will remove it for now since it works and is easier to use. When we want it in the future we can still ad it.

Reply:
I think it's still best to pass the ganach cmd as an env variable because that's easiest to configure in CI configs (like in your github one)

Reply:
this was for local testing reasons when i wanted to run ganache separately. i'm fine with removing that option.

Reply:
Ok good